daisy_workflows/image_test/oslogin-ssh/oslogin_slave_tester.sh

```diff
@@ -35,32 +35,34 @@ exec {fd_stdout}>&1
 # Redirect STDOUT to STDERR
 exec 1>&2
 # Redirect STDERR to Serial port
-exec 2>/dev/ttyS0
+sudo exec 2>/dev/ttyS0
 
 set -eux
 
-echo "Executing \$0 \$@"
+echo "Executing $0 $@"
 
 KEY=daisy-key
 
-case \$1 in
+case $1 in
   add_key)
-    if [ ! -f \${KEY}.pub ]; then
-      ssh-keygen -t rsa -N '' -f daisy-key -C "\$(uname -n)"
+    if [ ! -f ${KEY}.pub ]; then
+      ssh-keygen -t rsa -N '' -f daisy-key -C "$(uname -n)"
     fi
-    gcloud compute os-login ssh-keys add --key-file=\${KEY}.pub
-    cat \${KEY}.pub >&"\$fd_stdout"
+    cat ${KEY}.pub >&"$fd_stdout"
+    NKEYS=$(gcloud compute os-login ssh-keys list | grep -v FINGERPRINT | wc -l)
+    echo "Total number of keys currently on os-login: $NKEYS"
```

Comment:
I added this echo to be able to check if os-login's keys is a crowded place or not. Specially if this number grows after a while